src/test/java/seedu/duke/command/PinCommandTest.java

```diff
@@ -1,10 +1,78 @@
 package seedu.duke.command;
 
+import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import seedu.duke.data.notebook.Note;
+import seedu.duke.data.notebook.Notebook;
```

Comment:
Perhaps follow standardization after Code Review 1.0, put an empty line between group of imports. Group the import by usage/type.

Reply:
Thanks. Have changed it